### PR TITLE
[Feature] Update Footer Design #2

### LINKED ISSUE #2: [Feature] Update Footer Design
### Description

Update Footer Design, please make sure use Shadcn + tailwind css for design

### Screenshots

_No response_

### Additional Information

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Prithwi32**
## Fixes Issue
[Feature] Update Footer Design

 Closes #2 

## Changes proposed
Alignment of icons and text size and colour changes accordingly

## Screenshots
image

### CODE REVIEW

File: pnpm-lock.yaml
Comment (AbhiPatel10) [inline pnpm-lock.yaml:1]: Looks Good to me, Can you please just remove this file. (pnpm-lock.yml)

Comment (Prithwi32) [reply pnpm-lock.yaml:1]: > Looks Good to me, Can you please just remove this file. (pnpm-lock.yml)

@AbhiPatel10 ..could you check it now

**Comment by AbhiPatel10** [comment]
Do not remove the whole pnpm-locks file Just remove the change of that file.

If you are facing any issue then please create a new PR with only Footer Changes

**Comment by Prithwi32** [comment]
> Do not remove the whole pnpm-locks file Just remove the change of that file.
> 
> If you are facing any issue then please create a new PR with only Footer Changes

okay

**Comment by Prithwi32** [comment]
@AbhiPatel10 could you please check out know, I added as per main file

**Review by AbhiPatel10 — APPROVED** [review]
Looks good to me

**Comment by AbhiPatel10** [comment]
@Prithwi32 , Thank you so much for your contributions.

**Pull request merged**